docs/deploy-teleirc.rst

```diff
@@ -0,0 +1,95 @@
+#####################
+How to deploy Teleirc
+#####################
+
+There are several ways to deploy Teleirc persistently.
+This page offers suggestions on possible deployment options.
+
+
+*******
+systemd
+*******
+
+The **recommended deployment method** is with `systemd <https://en.wikipedia.org/wiki/Systemd>`_.
+This method requires a basic understanding of systemd unit files.
+Create a unique systemd service for each Teleirc instance.
```

Comment:
Inspired https://github.com/RITlug/teleirc/issues/123

Reply:
@thenaterhood Thanks for pointing it out, I learned something new here. :+1: